packages/types/src/types/models-extra/dashboard-item/charts/common.ts

```diff
@@ -26,7 +26,70 @@ type YAxisDomainEntry = {
   max?: number;
 };
 export type YAxisDomain = {
+  /**
+   * @description
+   * Domain configuration for the maximum y axis value.
+   * eg.
+   * // Cannot be higher than 100
+   * {
+   *   type: number
+   *   value: 100
+   * }
+   *
+   * // Clamp between 50, the maximum data value, and 100
+   * {
+   *   type: clamp
+   *   min: 50
+   *   max: 100
+   * }
+   *
+   * // Scale the y axis to always be 10% higher than the maximum data value
+   * {
+   *   type: scale
+   *   value: 1.1
+   * }
+   *
+   * // 20 greater than the maximum data value
+   * {
+   *   type: string
+   *   value: datamax + 20
+   * }
+   *
+   * For more options see: https://recharts.org/en-US/api/YAxis#domain
+   */
```

Comment:
This is where `@example` shines! But regardless, thank you for documenting these here 🏆

```suggestion
  /**
   * @description Domain configuration for the maximum y-axis value.
   *
   * @example Cannot be higher than 100
   * {
   *   type: number
   *   value: 100
   * }
   *
   * @example Clamp between 50, the maximum data value, and 100
   * {
   *   type: clamp
   *   min: 50
   *   max: 100
   * }
   *
   * @example Scale the y-axis to always be 10% higher than the maximum data value
   * {
   *   type: scale
   *   value: 1.1
   * }
   *
   * @example 20 greater than the maximum data value
   * {
   *   type: string
   *   value: datamax + 20
   * }
   *
   * @see https://recharts.org/en-US/api/YAxis#domain
   */
```

Reply:
We need to use the `@description` tag for the examples to show up in the hover tooltip in the json editor ui, otherwise would definitely go with your suggestion